source/initial-dns-seedlist-discovery/tests/uri-with-auth.json

```diff
@@ -9,11 +9,9 @@
     "localhost:27018",
     "localhost:27019"
   ],
-  "options": {
-    "auth": {
-      "user": "auser",
-      "password": "apass"
-    }
+  "parsed_options": {
+    "user": "auser",
+    "password": "apass"
```

Comment:
Can you add this new group and what it is meant for to the [README.rst](https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/tests/README.rst#test-format-and-use) too?